bus_board.rb

```diff
@@ -0,0 +1,56 @@
+require 'net/http'
+require 'uri'
+require 'json'
+
+
+class BusStop
+
+  def initialize(id)
+    @id = id
+    @buses_info = {}
+  end
+
+  def web_request()
+    require 'net/http'

```

Comment:
Do you need this? It's also at the top of the file and you'd normally try to keep all `require`s there.